### ISSUE TITLE
[regression] Error upon submitting new metadata

### ISSUE THREAD

**Issue by proycon** [proycon/flat#155]
Upon adding a new metadata field and saving the result, FLAT says:

```
Editor submission failed: FoLiA Document Server error: The server encountered an unexpected condition which prevented it from fulfilling the request.
```

The query doesn't seem to make it to foliadocserve, FLAT returns a HTTP 403 with the above message.

**Comment by proycon** [comment]
Issue reproduces also with the current development builds of FLAT and foliadocserve

**Comment by proycon** [comment]
Fixed in foliadocserve v0.7.1
